components/tech/Technology.vue

```diff
@@ -4,7 +4,7 @@
       <v-stack md-direction="column" justify="space-between" align="center">
         <v-stack direction="column" class="technology-hero-section">
           <app-section-descriptor title="Technology" />
-          <v-heading :level="1">
+          <v-heading :level="1" transform="capitalize">
```

Comment:
Note that `text-transform: capitalize` does not follow language-specific rules, like not capitalising articles (a, an, the) and conjunctions (and). It's probably better for us to do this manually if we need to be more specific.

+ @shaloo How do you recommend we capitalise our titles?

Reply:
Sure

Reply:
Resolved at https://github.com/arcana-network/website/pull/123/commits/92dfe59e4d2acb8b046d757c7ca1fb0f14b290a1